js/Track/Model/File/BED.js

```diff
@@ -19,15 +35,27 @@ Genoverse.Track.Model.File.BED = Genoverse.Track.Model.File.extend({
           color = this.scoreColor(isNaN(score) ? 1000 : score);
         }
 
-        this.insertFeature({
-          chr             : chr,
-          start           : parseInt(fields[1], 10),
-          end             : parseInt(fields[2], 10),
-          id              : chr + ':' + fields[1] + '-' + fields[3],
-          label           : fields[3],
-          color           : color,
-          originalFeature : fields
-        });
+        if(len == 12){ //subfeatures present
+          feature.blockCount = parseInt(fields[9],10);
+          var subfeatures = [];
+          var blockSizes = fields[10].split(",");
+          var blockStarts = fields[11].split(",");
+          blockSizes.pop();
+          blockStarts.pop();
```

Comment:
Hmm, the example here https://genome.ucsc.edu/FAQ/FAQformat#format1 is 
```
track name=pairedReads description="Clone Paired Reads" useScore=1
chr22 1000 5000 cloneA 960 + 1000 5000 0 2 567,488, 0,3512
chr22 2000 6000 cloneB 900 - 2000 6000 0 2 433,399, 0,3601
```
which has a trailing comma on blockSize but not on blockStart. I think it would be better to have them both defined as 
```
fields[10 or 11].split(",").filter(function (n) { return n; })
```
That way we can be sure of not losing the last value if it exists